### DISCUSSION TITLE
hover of element isn't removed when options of select is hidden and shown

### DISCUSSION THREAD

**Discussion by yureckey** [quasarframework/quasar#14310] [General - Components / Directives / etc] [answered]
### What happened?

I have select with several toggle elements, and user can check/uncheck multiple elements. When user checks some, then hides selector, then opens selector again, there is one of elements still in hovered state - which confuses user.
(hovered is the element user selected first, but there could be no indication of that with enabled "hide-selected")

### What did you expect to happen?

hovering isn't shown when selector is hidden and shown again

### Reproduction URL

https://codepen.io/yureckey/pen/MWGgzZa

### How to reproduce?

1. go to the link
2. click "Filter Values"
3. check "Value D", then "Value C", then "Value B"
4. click anywhere to hide selector.
5. click "Filter Values" again.
6. see that "Value D" is in hovered state, which look weird for user

### Flavour

Quasar CLI with Vite (@quasar/cli | @quasar/app-vite)

### Areas

Components (quasar)

### Platforms/Browsers

Chrome

### Quasar info output

```Shell
latest
```

**Comment by metalsadman** [comment]
it's not a hover, rather a focus state, it focuses on the first element of your v-model, which is D in your case.

**Comment by yureckey** [comment]
doesn't matter if it's hover or focus - it should be reset after options popup is hidden

**Comment by metalsadman** [comment, marked as answer]
use a template ref on your qselect, and use `setOptionIndex` if you don't want this behavior. https://quasar.dev/vue-components/select#qselect-api
![Screen Shot 2022-09-02 at 2 19 11 PM](https://user-images.githubusercontent.com/31038838/188071461-1d1982cc-f1eb-4288-8aff-ddf20304a50a.png)

https://codepen.io/metalsadman/pen/YzLzwGj

**Reply by yureckey** [reply]
Yes, that would work in my case, thank you.
